scala/private/toolchain_deps/toolchain_deps.bzl

```diff
@@ -0,0 +1,18 @@
+load("@io_bazel_rules_scala//scala:providers.bzl", "DepsInfo")
+
+def _log_required_provider_id(target, toolchain_type_label, provider_id):
```

Comment:
isn't it misleading? it's called log but you're actually failing

Reply:
I agree.